youtube/guide-for-beginners.md

```diff
@@ -1,137 +1,96 @@
-# Notice! ⚠️ [From now on use the new RvX Manager created specially for ReVanced Extended patches.](https://github.com/inotia00/revanced-manager/releases/latest)
+# Notice: ⚠️ [From now on use the new RVX Manager created specially for ReVanced Extended patches.](https://github.com/inotia00/revanced-manager/releases/latest)
 
-# About this Guide:
-‎ 
-[‎ ](LineBreak)
-‎
-### **Originally created by /u/KobeW50**.
+.
 
-‎[‎ ](LineBreak)

```

Comment:
I removed the invisible characters because the hyperlink shows as a small "2" in Sync. I also changed the info headers to just bold text for easier reading. This also makes it so that the line break isn't needed (except after the giant notice on top of the post)